aiven/client/cli.py

```diff
@@ -2020,6 +2049,14 @@ def service__task_create(self):
     )
     def service__topic_create(self):
         """Create a Kafka topic"""
+
+        tags = list(map(parse_tag_str, self.args.topic_option_tag))
+        tag_keys = list(map(lambda d: d.get("key"), tags))
+        if len(set(tag_keys)) != len(tag_keys):
```

Comment:
This can be done with a [Counter](https://docs.python.org/3/library/collections.html#collections.Counter):

```python
repeated_keys = [key for key, count in Counter(tag_keys).items() if count > 1]
```